### PR TITLE
Update Tizen RT Guide.

### PR DESCRIPTION
**Pull request by Hyunjun85**
Update for Tizen RT IDE latest version.

### CODE REVIEW

File: docs/application/tizen-studio/rt-ide/rt-overview.md [hunk 1]
```diff
@@ -1,12 +1,23 @@
-# RT IDE
+# Tizen RT
 
-The Tizen Studio for RT is an IDE that provides a lightweight RTOS (real-time operating system)-based application development environment that helps you develop, build, flash, and debug applications in Ubuntu.
+Tizen has been commercialized in smart TV, smartphone, wearable devices (Gear S, Gear Fit), and smart home appliances. However, low-end and low-cost IoT devices, such as home appliances without display and wearable bands with a small LCD, have received less attention. The goal of TizenRT is to extend the Tizen platform device coverage to these kind of low-end devices.
```
Comment (surbhig12) [inline docs/application/tizen-studio/rt-ide/rt-overview.md]: Minor changes, please rephrase to:

Tizen has commercialized in smart TV, smartphone, wearable devices (Gear S, Gear Fit), and smart home appliances. However, low-end and low-cost IoT devices, such as home appliances without display and wearable bands with a small LCD, have received less attention. The objective of Tizen RT is to extend the Tizen platform device coverage to these kind of low-end devices.

File: docs/application/tizen-studio/rt-ide/rt-overview.md [hunk 2]
```diff
@@ -1,12 +1,23 @@
...
 
-To develop your applications, use the set of tools provided by the Tizen Studio for RT:
+Tizen RT is an RTOS-based lightweight platform.
```
Comment (surbhig12) [inline docs/application/tizen-studio/rt-ide/rt-overview.md]: Tizen RT is an RTOS-based lightweight platform.

Please rephrase to:

Tizen RT is an RTOS-based lightweight platform. The following are the hardware and environmental requirements:

File: docs/application/tizen-studio/rt-ide/rt-overview.md [hunk 3]
```diff
@@ -1,12 +1,23 @@
...
 
--   [Getting Started](getting-started/overview.md)
--   [SmartThings&trade; Extension](smartthings-ext/overview.md)
+* Hardware targets
+Cortex-M/R processors with MPU, less than 2 MB RAM, and less than 16 MB Flash.
```
Comment (surbhig12) [inline docs/application/tizen-studio/rt-ide/rt-overview.md]: Cortex-M/R....

This text must appear in the next line.

File: docs/application/tizen-studio/rt-ide/rt-overview.md [hunk 4]
```diff
@@ -1,12 +1,23 @@
...
 
-## Related Information
-* Dependencies
- - Ubuntu Only
+* Development environment
+	- Tizen RT Studio for SmartThings
+	Tizen RT Studio 2.0 (or later) is used as an IDE for developing IoT applications with the Tizen IoT Platform and ARTIK 053(s).
+
```
Comment (surbhig12) [inline docs/application/tizen-studio/rt-ide/rt-overview.md]: Please rephrase to:

Tizen RT Studio for SmartThings: Tizen RT Studio version 2.0 (or later) is used as an IDE for developing IoT applications with the Tizen IoT Platform and ARTIK 053(s).

File: docs/application/tizen-studio/rt-ide/rt-overview.md [hunk 5]
```diff
@@ -1,12 +1,23 @@
...
+	- IoT Setup Wizard
+	The TizenRT Setup Wizard is a computer application for easily setting up Tizen IoT on hardware targets. In 4. Flash the project, flashing a platform image into ARTIK 053(s) is only supported with the IoT Setup Wizard in a Linux computer environment.
```
Comment (surbhig12) [inline docs/application/tizen-studio/rt-ide/rt-overview.md]: Please rephrase to:

IoT Setup Wizard: The Tizen RT Setup Wizard is a computer application for easily setting up Tizen IoT on hardware targets. In the Tizen OS version 4.0, flashing a platform image into ARTIK 053(s) is only supported with the IoT Setup Wizard in a Linux computer environment.

File: docs/application/tizen-studio/rt-ide/rt-overview.md [hunk 6]
```diff
@@ -1,12 +1,23 @@
...
+
+To get started with developing your own Tizen IoT applications:
```
Comment (surbhig12) [inline docs/application/tizen-studio/rt-ide/rt-overview.md]: Please rephrase to:

To start developing your Tizen IoT applications:

File: docs/application/tizen-studio/rt-ide/rt-overview.md [hunk 7]
```diff
@@ -1,12 +1,22 @@
...
- - Ubuntu Only
+* Development environment:
+	- Tizen RT Studio for SmartThings:
+	Tizen RT Studio versionn 2.0 (or later) is used as an IDE for developing IoT applications with the Tizen IoT Platform and ARTIK 053(s).
+
+	- IoT Setup Wizard:
+	The Tizen RT Setup Wizard is a computer application for easily setting up Tizen IoT on hardware targets. In the Tizen OS version 4.0, flashing a platform image into ARTIK 053(s) is only supported with the IoT Setup Wizard in a Linux computer environment.
+
+To get start developing your Tizen IoT applications:
+1. [Install the Tizen Studio for RT](rt-install.md)
```
Comment (mijin85cho) [inline docs/application/tizen-studio/rt-ide/rt-overview.md]: Also, please check the links here. These links are broken.

**Comment by mijin85cho** [comment]
@Hyunjun85 Hello, Hyunjun85. I can see there are not-updated parts in the file after the review. Would you please check the broken link, too, and update them?

**Comment by mijin85cho** [comment]
@Hyunjun85 Hello, Hyunjun85. The broken links update requested are not fixed yet. I'm worrying if this PR shouldn't be merged before 03.30, the target date of release. Please check the comment and update the PR.

**Comment by Hyunjun85** [comment]
Hello, @mijin85cho.
I had updated some broken links. 
Could you review again?

**Pull request merged**